src/explore-education-statistics-admin/src/pages/release/data/components/ReorderFiltersList.tsx

```diff
@@ -0,0 +1,242 @@
+  SubjectMeta,
+} from '@common/services/tableBuilderService';
+import orderBy from 'lodash/orderBy';
+import React, { useEffect, useState } from 'react';
+
+// EES-1243 - remove this when the backend is done
+const testSubjectMeta: SubjectMeta = {
+  filters: {
+    Category1: {
+      id: 'category-1-id',
+      legend: 'Category 1',
+      name: 'category-1',
+      options: {
+        Category1Group1: {
+          id: 'category-1-group-1-id',
+          label: 'Category 1 Group 1',
+          options: [
+            {
+              label: 'Category 1 Group 1 Item 1',
+              value: 'category-1-group-1-item-1',
+              id: 'category-1-group-1-item-1-id',
+            },
+          ],
+          order: 1,
+        },
+        Category1Group2: {
+          id: 'category-1-group-2-id',
+          label: 'Category 1 Group 2',
+          options: [
+            {
+              label: 'Category 1 Group 2 Item 1',
+              value: 'category-1-group-2-item-1',
+              id: 'category-1-group-2-item-1-id',
+            },
+            {
+              label: 'Category 1 Group 2 Item 2',
+              value: 'category-1-group-2-item-2',
+              id: 'category-1-group-2-item-2-id',
+            },
+            {
+              label: 'Category 1 Group 2 Item 3',
+              value: 'category-1-group-2-item-3',
+              id: 'category-1-group-2-item-3-id',
+            },
+          ],
+          order: 0,
+        },
+        Category1Group3: {
+          id: 'category-1-group-3-id',
+          label: 'Category 1 Group 3',
+          options: [
+            {
+              label: 'Category 1 Group 3 Item 1',
+              value: 'category-1-group-3-item-1',
+              id: 'category-1-group-3-item-1-id',
+            },
+            {
+              label: 'Category 1 Group 3 Item 2',
+              value: 'category-1-group-3-item-2',
+              id: 'category-1-group-3-item-2-id',
+            },
+          ],
+          order: 2,
+        },
+      },
+      order: 1,
+    },
+    Category2: {
+      id: 'category-2-id',
+      legend: 'Category 2',
+      name: 'category-2',
+      options: {
+        Category2Group1: {
+          id: 'category-2-group-1-id',
+          label: 'Category 2 Group 1',
+          options: [
+            {
+              label: 'Category 2 Group 1 Item 1',
+              value: 'category-2-group-1-item-1',
+              id: 'category-2-group-1-item-1-id',
+            },
+            {
+              label: 'Category 2 Group 1 Item 2',
+              value: 'category-2-group-1-item-2',
+              id: 'category-2-group-1-item-2-id',
+            },
+          ],
+          order: 0,
+        },
+      },
+      order: 0,
+    },
+  },
+  indicators: {},
+  locations: {},
+  timePeriod: {
+    hint: '',
+    legend: '',
+    options: [],
+  },
+};
+
+interface UpdatedFilter {
+  id?: string;
+  filterGroups: { id?: string; filterItems: (string | undefined)[] }[];
```

Comment:
It's because I've had to put ids as optional on the type for filters as it doesn't exist yet, and this is an array of those ids. I'll add a note that it can be tidied up later.

Reply:
Gotcha!  Thanks :)